hostName: vicon
numConfigurations: 1
configurations:
  "0":
    numPoints: 4
    points:
      "0": [-0.023396,-0.019762,0.019306]
      "1": [0.012398,0.011179,0.027415]
      "2": [0.027939,-0.021060,0.019083]
      "3": [-0.016644,0.026118,0.018928]
numObjects: 2
objects:
  "0":
    name: crazyflie1
    configuration: 0
    initialPosition: [0, 0, 0]
    initialYaw: 0
    maxV: 1.0
    maxW: 3.0
  "1":
    name: crazyflie2
    configuration: 0
    initialPosition: [0, -0.75, 0]
    initialYaw: 0
    maxV: 1.0
    maxW: 3.0
  "2":
    name: crazyflie3
    configuration: 0
    initialPosition: [0, 0.75, 0]
    initialYaw: 0
    maxV: 1.0
    maxW: 3.0
